src/shared-components/immersiveModal/index.tsx

```diff
@@ -43,7 +43,7 @@ type ImmersiveModalProps = {
   /**
    * The title of the modal
    */
-  title?: string;
+  title?: string | React.ReactNode;
```

Comment:
We have one instance to cover that is:

```
<React.Fragment>
   Some text here
   <span>Some emoji here</span>
<React.Fragment>
```

Reply:
`type React.ReactNode` includes string do you think we can remove string at the beginning

Reply:
Went with `JSX.Element` instead to narrow the type more from `ReactNode`. Updated in PocketDerm as well without errors.